src/vue/Components/Password/View.vue

```diff
@@ -0,0 +1,311 @@
+<template>
+    <div class="item-menu password-view">
+        <div class="password-header">
+            <div class="left-space"/>
+            <div class="badge-container">
+                <icon :class="securityClass" icon="shield-alt" font="solid"/>
+                <icon class="favorite" @click="updateFavorite()" icon="star" :font="favoriteIconSolid"/>
+                <icon :icon="sharedIcon" font="solid"/>
+            </div>
+            <icon :icon="actionIcon" @click="toggleAction()" :class="actionClassList"/>
```

Comment:
Passwords have an [editable](https://git.mdns.eu/nextcloud/passwords/wikis/Developers/Api/Password-Api#the-password-object) property which can be false if the password is shared. In that case there should not be an edit button.

Reply:
I changed this for the edit button. But the favorite action is still allowed on shared elements or do I need to check the edited flag as well for setting the favorite?

Reply:
favorite is still allowed because favorites are per-user. Only fields that are shared between everyone will cause an error.

Reply:
ok perfect then this is also resolved 😉